Commons Box Minecraft/src/main/java/plugily/projects/commonsbox/minecraft/serialization/InventorySerializer.java

```diff
@@ -54,7 +54,17 @@ public static boolean saveInventoryToFile(JavaPlugin plugin, Player player) {
       invConfig.set("ExperienceProgress", player.getExp());
       invConfig.set("ExperienceLevel", player.getLevel());
       invConfig.set("Current health", player.getHealth());
-      invConfig.set("Max health", VersionUtils.getMaxHealth(player));
+
+      int max_health = VersionUtils.getMaxHealth(player);
+
+      PotionEffect effect = entity.getPotionEffect(PotionEffectType.HEALTH_BOOST);
+      if (effect != null) {
+        // Health boost effect has a base of 2 extra hearts then adds for 2 hearts for every level beyond
+        // 2 hearts (per level) is 4 half hearts (MAX_HEALTH is stored as half hearts)
```

Comment:
short the description to something like (one line) health boost effect gives +2 hearts per level, health is counted in half hearts so amplifier * 4